bndtools.core/src/bndtools/launch/api/AbstractLaunchShortcut.java

```diff
@@ -102,8 +125,8 @@ public void launch(IEditorPart editor, String mode) {
     }
 
     @SuppressWarnings("unused")
-    protected void launchJavaElement(IJavaElement element, String mode) throws CoreException {
-        launch(element.getJavaProject().getProject().getFullPath(), mode);
+    protected void launchJavaElements(List<IJavaElement> elements, String mode) throws CoreException {
+        launch(elements.iterator().next().getJavaProject().getProject().getFullPath(), mode);
```

Comment:
Up above you used `elements.get(0).getJava...`. Here it is `elements.iterator().next().getJava...`. Lets make them consistent. Since `elements` is a list, lets use `get(0)` here to match the code above.